src/openrct2/interface/Window_internal.cpp

```diff
@@ -4,13 +4,13 @@
 
 void rct_window::SetLocation(int32_t newX, int32_t newY, int32_t newZ)
 {
-    window_scroll_to_location(this, newX, newY, newZ);
+    window_scroll_to_location(this, { newX, newY, newZ });
     flags &= ~WF_SCROLLING_TO_LOCATION;
 }
 
 void rct_window::ScrollToViewport()
 {
-    int32_t newX = LOCATION_NULL, newY = LOCATION_NULL, newZ = LOCATION_NULL;
+    CoordsXYZ newCoords = {};
```

Comment:
Move this to just above the `if (viewport_focus_sprite.type & VIEWPORT_FOCUS_TYPE_SPRITE)` statement.

Merge the `rct_window* mainWindow;` declaration with the definition on line 38.